gateway/config/staging.lua

```diff
@@ -0,0 +1,6 @@
+return {
+    master_process = 'on',
+    lua_code_cache = 'on',
+    configuration_loader = 'lazy',
+    configuration_cache = 0,
```

Comment:
Why not `os.getenv('APICAST_CONFIGURATION_CACHE') or 0` ?

Reply:
I think our staging should always reload configuration. And because of the order of `options` and `context` in the `start.lua` the CLI options take precedence. So setting `APICAST_CONFIGURATION_CACHE` env would override this anyway.

That makes the production.lua `os.getenv` a bit redundant, but I wanted it there as an example of what can be done. But if it is confusing I could change/remove it.